lib/charms/ip_router_interface/v0/ip_router_interface.py

```diff
@@ -135,7 +135,7 @@ def _action_request_network(self, event: ops.ActionEvent):
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 2
+LIBPATCH = 3
 
```

Comment:
Does the top level docstring properly represent how the lib is used with those new changes?

Reply:
Now it does